hugo.mk

```diff
@@ -6,10 +6,10 @@ DRAFT_FLAGS ?= --buildDrafts --verboseLog=true -v
 PROD_FLAGS ?= -s .
 S3_FLAGS ?= --acl-public --delete-removed --no-progress --no-mime-magic --guess-mime-type
 INTERMEDIATE_STEPS ?= echo nothing
-STAGING_BUCKET ?= stage.mediathread.info
+STAGING_URL=https://mediathreadinfo.stage.ctl.columbia.edu/
+PROD_URL=https://mediathread.info
+STAGING_BUCKET ?= mediathreadinfo.stage.ctl.columbia.edu
```

Comment:
If we try to keep site-specific stuff in Makefile, that allows our general makefiles like `hugo.mk`, etc, to be more easily updated automatically from the ccnmtl/makefiles repo.

Reply:
ah got it, i'll bounce it back